### PR TITLE
OCPBUGS-6004: filter ovn fd69::2 ip from list of ips in case ovn

### PR DESCRIPTION
**Pull request by tsorya**
Adding network type in order to be able to filter ovn ipv6 ip

### CODE REVIEW

**Comment by tsorya** [comment]
Looks good

Jan 16 15:33:25 02-00-00-4f-27-17 bash[18853]: time="2023-01-16T15:33:25Z" level=debug msg="Ignoring filtered address fd69::2/125"

File: pkg/utils/addresses.go [hunk 1]
```diff
@@ -241,6 +251,7 @@ func addressesDefaultInternal(preferIPv6 bool, af AddressFilter, getAddrs addres
 			})
 		}
 	}
+	fmt.Println(addrs)
```
Comment (cybertron) [inline pkg/utils/addresses.go:254]: Leftover debug statements? We typically use the log module for messages we want to keep.

File: pkg/utils/addresses.go [hunk 2]
```diff
@@ -277,6 +288,7 @@ func addressesDefaultInternal(preferIPv6 bool, af AddressFilter, getAddrs addres
 			foundv4 = true
 		}
 	}
+	fmt.Println(matches)
```
Comment (cybertron) [inline pkg/utils/addresses.go:291]: Ditto

**Review by cybertron — COMMENTED** [review]
LGTM modulo a couple of printfs that should either be removed or converted to log calls.

File: pkg/utils/addresses.go (hunk 1 above)
Comment (tsorya) [reply pkg/utils/addresses.go:254]: Yep, removing

**Review by cybertron — COMMENTED** [review]
/lgtm

**Comment by cybertron** [comment]
/hold

We should resolve the bug situation with this.

**Pull request merged**